libraries/lib-datahandler/src/main/thrift/moderation.thrift

```diff
@@ -46,15 +47,19 @@ struct ModerationRequest {
 
     // Moderation request
     10: required i64 timestamp,
+    37: optional i64 timestampOfDecision,
     11: required string documentId,
     12: required DocumentType documentType,
     13: optional string requestingUser,
+    36: optional UserGroup requestingUserGroup,
     14: optional set<string> moderators,
     15: optional string documentName,
     16: required ModerationState moderationState,
     17: optional string reviewer,
     18: required bool requestDocumentDelete,
-    19: optional string requestingUserDepartment;
+    19: optional string requestingUserDepartment,
```

Comment:
This is the field that should be filled. The field has already existed, but it was only set for license moderation requests - that's kind of a bug, I think. `ModerationDatabaseHandler.createStubRequest()` looks like a good place to set the field and ensure that all moderation requests have it